src/main/java/com/google/sps/model/AuthenticatedHttpServlet.java

```diff
@@ -26,6 +29,24 @@
   // Error message if user is not authenticated
   protected static final String ERROR_403 = "Authentication tokens not present / invalid";
 
+  private final AuthenticationVerifier authenticationVerifier;
+
+  /** Create AuthenticatedHttpServlet with default implementations of the AuthenticationVerifier */
+  public AuthenticatedHttpServlet() {
+    super();
+    authenticationVerifier = new AuthenticationVerifierImpl();
+  }
+
+  /**
+   * Create AuthenticatedHttpServlet with an explicit implementation of the AuthenticationVerifier
+   *
+   * @param authenticationVerifier implementation of the AuthenticationVerifier
+   */
+  public AuthenticatedHttpServlet(AuthenticationVerifier authenticationVerifier) {
+    super();
```

Comment:
This [link](https://stackoverflow.com/questions/4090834/when-do-i-use-super) might provide some insight to using the super() keyword.

Reply:
Interesting - thanks